src/Illuminate/Support/helpers.php

```diff
@@ -883,3 +883,15 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('is_winnt')) {
+    /**
+     * Determine whether the current envrionment is Windows NT based.
+     *
+     * @return bool
+     */
+    function is_winnt()
+    {
+        return substr(strtoupper(php_uname('s')), 0, 3) === 'WIN';
```

Comment:
If you decide to keep this method:
* `is_winnt` function name might be improved, as it doesn't match NT specifically ("WINNT", "WIN32", "Windows", per [this question on SO](http://stackoverflow.com/questions/738823/possible-values-for-php-os)). How about `is_windows`?
* looks better to me: `strtoupper(substr(php_uname('s'), 0, 3)) === 'WIN'`
* you could even do: `strncasecmp(php_uname('s'), 'WIN', 3) === 0`